Ch03/Ch03.md

```diff
@@ -336,7 +335,7 @@ reference. Alice and Henry (and Edith) only hold the _identifier_ for their pare
 How is this model reflected in our code? Let's look at Listing 3.4 to see that (again, with some information redacted to make it easier
 to focus on the relevant parts).
 
-```
+```{caption="Child class representing the model" .json}
```

Comment:
This will create a new listing number, no? Where is that refered?

Reply:
The Listing number *already existed* (3.4). It was just missing the caption.